themes/default/content/docs/languages-sdks/yaml/yaml-language-reference.md

```diff
@@ -135,7 +135,8 @@ resources:
 To create an explicit provider instance, preferably with a specific version, use the [`resources`](#resources) section and prefix the name of the provider with `pulumi:providers` which will the value of the `type` property.
```

Comment:
```suggestion
To create an explicit provider instance, preferably with a specific version, use the [`resources`](#resources) section to declare a `provider`, prefixing the name with `pulumi:providers`.
```

Reply:
Thanks for spotting! I edited it as: 

> For the `type` property, prefix the name of the provider with `pulumi:providers`.

I believe this is slightly more clear but feel free to disagree, happy to go with your original suggestion.